reporter/src/main/kotlin/reporters/ReportTableModelMapper.kt

```diff
@@ -39,15 +39,14 @@ import com.here.ort.reporter.reporters.ReportTableModel.SummaryTable
 
 private fun Collection<ResolvableIssue>.filterUnresolved() = filter { !it.isResolved }
 
-private fun Excludes.scopeExcludesByName(scopeNames: Collection<String>): Map<String, List<ScopeExclude>> =
-    scopeNames.associateWith { scopeName -> findScopeExcludes(scopeName) }
-
-private fun Project.getScopesForDependencies(): Map<Identifier, Set<String>> {
-    val result = mutableMapOf<Identifier, MutableSet<String>>()
+private fun Project.getScopesForDependencies(excludes: Excludes): Map<Identifier, Map<String, List<ScopeExclude>>> {
+    val result = mutableMapOf<Identifier, MutableMap<String, List<ScopeExclude>>>()
 
     scopes.forEach { scope ->
         scope.collectDependencies().forEach { dependency ->
-            result.getOrPut(dependency, { mutableSetOf() }).add(scope.name)
+            result
```

Comment:
Nit: Formatting is bit odd here, our usual formatting would be

            result.getOrPut(dependency, { mutableMapOf() })
                .getOrPut(scope.name, { excludes.findScopeExcludes(scope.name) })

Reply:
Done. However, I wonder if it would make sense to allow both ways of formatting this as I think
separate lines can be more readable in some cases so we could leave the freedom of choice up to the author. What do you think?

Reply:
I believe we should stick to one style of formatting for the whole project, and ideally enforce that style e.g. via custom detekt rules. But you're right, as long as there are no matching rules (and also IDE inspection does not complain), there is some freedom for the author. Which simply means we should implement a rule to restrict that freedom :laughing: (just kidding)

Reply:
yes, imo we should try to eliminate formatting discussions from the reviews as much as possible which automated checks plus some agreement would actually solve.